account.go

```diff
@@ -61,37 +61,54 @@ const (
 
 // AccountParams are the parameters allowed during account creation/updates.
 type AccountParams struct {
-	Params
-	Country, Email, DefaultCurrency, Statement, BusinessName, BusinessUrl,
-	BusinessPrimaryColor, SupportPhone, SupportEmail, SupportUrl,
-	FromRecipient, PayoutStatement string
-	ExternalAccount                      *AccountExternalAccountParams
-	LegalEntity                          *LegalEntity
-	PayoutSchedule                       *PayoutScheduleParams
-	DebitNegativeBal, NoDebitNegativeBal bool
-	TOSAcceptance                        *TOSAcceptanceParams
-	Type                                 AccountType
+	Params               `form:"*"`
+	Country              string                        `form:"country"`
+	Email                string                        `form:"email"`
+	DefaultCurrency      string                        `form:"default_currency"`
+	Statement            string                        `form:"statement_descriptor"`
+	BusinessName         string                        `form:"business_name"`
+	BusinessUrl          string                        `form:"business_url"`
+	BusinessPrimaryColor string                        `form:"business_primary_color"`
+	SupportPhone         string                        `form:"support_phone"`
+	SupportEmail         string                        `form:"support_email"`
+	SupportUrl           string                        `form:"support_url"`
+	FromRecipient        string                        `form:"from_recipient"`
+	PayoutStatement      string                        `form:"payout_statement_descriptor"`
+	ExternalAccount      *AccountExternalAccountParams `form:"external_account"`
+	LegalEntity          *LegalEntity                  `form:"legal_entity"`
+	PayoutSchedule       *PayoutScheduleParams         `form:"payout_schedule"`
+	DebitNegativeBal     bool                          `form:"debit_negative_balances"`
+	NoDebitNegativeBal   bool                          `form:"debit_negative_balances,invert"`
+	TOSAcceptance        *TOSAcceptanceParams          `form:"tos_acceptance"`
+	Type                 AccountType                   `form:"type"`
 }
 
 // AccountListParams are the parameters allowed during account listing.
 type AccountListParams struct {
-	ListParams
+	ListParams `form:"*"`
 }
 
 // AccountExternalAccountParams are the parameters allowed to reference an
 // external account when creating an account. It should either have Token set
 // or everything else.
 type AccountExternalAccountParams struct {
-	Params
-	Account, Country, Currency, Routing, Token string
+	Params            `form:"*"`
```

Comment:
shouldn't the `form:"*"` be aligned with the other ones?

Reply:
So believe it or not, this is actually gofmt's preferred style.

I think it works by considering each field to be almost like a column in a table. It's not so important that each field is aligned with its like fields, but more that it's aligned with it's column number. So in this case it would be column no. 2 and aligned with the types of other fields because it doesn't have a type.